lib/screens/about/about.dart

```diff
@@ -0,0 +1,143 @@
+import 'dart:convert';
+import 'package:flutter/gestures.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:fogosmobile/actions/contributors_actions.dart';
+import 'package:fogosmobile/models/app_state.dart';
+import 'package:fogosmobile/models/contributor.dart';
+import 'package:fogosmobile/screens/about/contributor_item.dart';
+import 'package:fogosmobile/screens/components/fire_gradient_app_bar.dart';
+import 'package:fogosmobile/utils/uri_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:redux/redux.dart';
+
+
+class About extends StatefulWidget {
+  @override
+  _AboutState createState() => _AboutState();
+}
+
+class _AboutState extends State<About> {
+
+  List<Contributor> contributors = new List();
+
+  Widget contributorsWidget(List<Contributor> contributors) {
+    return new StoreConnector<AppState, AppState>(
+        converter: (Store<AppState> store) => store.state,
+        onInit: (store) {
+          store.dispatch(LoadContributorsAction());
+        },
+        builder: (BuildContext context, AppState state) {
+          if (state.hasContributors == true){
+            return new StoreConnector<AppState, List>(
+                converter: (Store<AppState> store) => store.state.contributors,
+                builder: (BuildContext context, List contributors) {
+                  if (contributors != null) {
+                    contributors = contributors;
```

Comment:
This can result in issues due to name ambiguity. I would either make it `this.contributors` or rename one of it.